src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -10,12 +10,17 @@
 
 	private DriverManager driverManager;
 
+
 	public SelectTestFigureOptionListener(DriverManager driverManager) {
 		this.driverManager = driverManager;
 	}
 
 	@Override
 	public void actionPerformed(ActionEvent e) {
-		FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+
+		if(e.getActionCommand().equals("Figure Joe 1"))
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.